loadbalancer_public.tf

```diff
@@ -53,78 +53,65 @@ resource "aws_security_group" "quortex_public" {
 }
 
 # Security group rules
-resource "aws_security_group_rule" "lb_public_http" {
+resource "aws_vpc_security_group_ingress_rule" "lb_public_http" {
   for_each = var.load_balancer_public_expose_http ? local.public_lb_allowed_ip_ranges : []
 
   description       = "Allow simple HTTP from whitelisted ip ranges only"
-  type              = "ingress"
   from_port         = 80
   to_port           = 80
-  protocol          = "tcp"
-  cidr_blocks       = [each.value]
+  ip_protocol       = "tcp"
+  cidr_ipv4         = each.value
   security_group_id = aws_security_group.quortex_public.id
 
-  lifecycle {
-    create_before_destroy = true
-  }
-
-  depends_on = [
-    aws_security_group_rule.lb_public_http_prefix_list
-  ]
+  tags = var.tags
```

Comment:
Do I need to propose a custom name for each resource type ?

Reply:
Is it really useful ?